clang/lib/CodeGen/CodeGenFunction.h

```diff
@@ -3353,6 +3353,17 @@ class CodeGenFunction : public CodeGenTypeCache {
                      SanitizerSet SkippedChecks = SanitizerSet(),
                      llvm::Value *ArraySize = nullptr);
 
+  /// Returns debug info, with additional annotation if enabled by
+  /// CGM.getCodeGenOpts().SanitizeAnnotateDebugInfo[CheckKindOrdinal].
+  llvm::DILocation *
+  SanitizerAnnotateDebugInfo(SanitizerKind::SanitizerOrdinal CheckKindOrdinal);
+
+  /// Converts the CFITypeCheckKind into SanitizerKind::SanitizerOrdinal and
+  /// llvm::SanitizerStatKind.
+  void ParseCFITypeCheckKind(CFITypeCheckKind TCK,
```

Comment:
if it's unused outside cpp, make it static threre

Reply:
Fixed in https://github.com/llvm/llvm-project/pull/139809/commits/fd3fd98add4cfadda22059dc6f175c08627ac282

Reply:
Sorry, I meant to ask this static usecase: https://llvm.org/docs/CodingStandards.html#id59